docs/osf-trivia-list/dasharo.md

```diff
@@ -147,6 +147,26 @@ There is also a libre, POWER9-based server/workstation Talos II by Raptor
 Computing Systems, which also do not use any binary blobs, however it is more
 expensive than x86 platforms.
 
+## How open is Dasharo?
+<!-- This answer needs improvements
+
+Place to describe openess score
+
+-->
+
+We are strong believers in Freedom and Open Source Software (FOSS), but we also
+have to deal with reality. Silicon Vendors and other providers in the firmware
+supply chain try to monopolize the ecosystem by including NDA procedures and
+binary blobs. Because of that, Dasharo has to provide seamless integration of
+Intel FSP, AMD AGESA and other binary components. Dasharo is needed for
```

Comment:
I know it's copied from a page, but we have a chance to revisit. Are we confident enough that Dasharo is **the only** `the only product that transparently leverages the Open Source Firmware`. This sounds like without Dasharo, the mission of liberating firmware ecosystem is doomed.

Reply:
@macpijan I don't consider those answers ready. Above, there is a comment that this needs improvement.